packages/link-modules/src/repositories/base.ts

```diff
@@ -0,0 +1,183 @@
+import { Context, DAL } from "@medusajs/types"
```

Comment:
todo: could you add a todo to remove that once this [one](https://github.com/medusajs/medusa/pull/4631) is merged